### PR TITLE
Prometheus client and server

### PR DESCRIPTION
**Pull request by Tezno**

### CODE REVIEW

File: docker/docker-compose-observability.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+version: "3"
+
+volumes:
+  prometheus_data: {}
```
Comment (RaboliotLeGris) [inline docker/docker-compose-observability.yml:4]: Can't we use a volume with a path ? Like it's done for the external services. This way, we could actually know where our data is.

File: docker/docker-compose-observability.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+services:
+  grafana:
+    image: grafana/grafana:latest
```
Comment (RaboliotLeGris) [inline docker/docker-compose-observability.yml:8]: I'm a bit sad that hadolint doesn't work for docker-compose file :(

Shouldn't we use a more precise tag ?

File: src/cmd/api/controllers/video_upload_handler.go [hunk 3]
```diff
@@ -18,7 +20,22 @@ type VideoUploadHandler struct {
 	AmqpClient clients.IAmqpClient
 }
 
+var (
```
Comment (RaboliotLeGris) [inline src/cmd/api/controllers/video_upload_handler.go:23]: an we regroup them in on var definition and have more accurate names ?

I'm a bit puzzled about those vars, shouldn't we declare them in a separate file ?

File: docker/docker-compose-observability.yml (hunk 1 above)
Comment (Tezno) [reply docker/docker-compose-observability.yml:4]: `The Prometheus image uses a volume to store the actual metrics. For production deployments it is highly recommended to use a named volume to ease managing the data on Prometheus upgrades.` from https://prometheus.io/docs/prometheus/latest/installation/

**Review by RaboliotLeGris — APPROVED** [review]

**Pull request merged**